packages/abi-coder/src/utils/constants.ts

```diff
@@ -10,18 +10,24 @@ export const RAW_PTR_CODER_TYPE = 'raw untyped ptr';
 export const RAW_SLICE_CODER_TYPE = 'raw untyped slice';
 export const BOOL_CODER_TYPE = 'bool';
 export const B256_CODER_TYPE = 'b256';
-export const B512_CODER_TYPE = 'struct B512';
-export const OPTION_CODER_TYPE = 'enum Option';
-export const VEC_CODER_TYPE = 'struct Vec';
-export const BYTES_CODER_TYPE = 'struct Bytes';
-export const STD_STRING_CODER_TYPE = 'struct String';
+export const B512_CODER_TYPE = 'struct std::b512::B512';
+export const OPTION_CODER_TYPE = 'enum std::option::Option';
+export const VEC_CODER_TYPE = 'struct std::vec::Vec';
+export const BYTES_CODER_TYPE = 'struct std::bytes::Bytes';
+export const STD_STRING_CODER_TYPE = 'struct std::string::String';
```

Comment:
Are these type formats the same for both `debug` and `release`?

Reply:
Yes, because now `--json-abi-with-callpaths` flag became the default behavior.

Reply:
Good to know - thanks.

Do we also want to update the associated typegen `swayType` (e.g. [B512Type](https://github.com/FuelLabs/fuels-ts/blob/3b62ef47a32129ae8b6a9d8296435bb1109051c3/packages/abi-typegen/src/abi/types/B512Type.ts#L4-L5))?

Reply:
Rephrasing: Type formats have never differed between `debug` and `release`, only when using the `--json-abi-with-callpaths` flag, which has now become the default behavior.